tests_oval_graph/test_arf_xml_parser/test_arf_xml_parser.py

```diff
@@ -47,6 +68,44 @@ def test_parsing_and_evaluate_scan_rule(rule_id, result):
     assert oval_tree.evaluate_tree() == result
 
 
+@pytest.mark.parametrize("rule_id, result", [
+    (
+        "xccdf_org.ssgproject.content_rule_accounts_passwords_pam_faillock_deny",
+        "false",
+    ),
+    (
+        "xccdf_org.ssgproject.content_rule_sshd_disable_gssapi_auth",
+        "false",
+    ),
+    (
+        "xccdf_org.ssgproject.content_rule_service_debug-shell_disabled",
+        "true",
+    ),
+    (
+        "xccdf_org.ssgproject.content_rule_mount_option_dev_shm_noexec",
+        "false",
+    ),
+    (
+        "xccdf_org.ssgproject.content_rule_audit_rules_unsuccessful_file_modification_creat",
+        "false",
+    ),
+    (
+        "xccdf_org.ssgproject.content_rule_audit_rules_file_deletion_events_rmdir",
+        "false",
+    ),
+    (
+        "xccdf_org.ssgproject.content_rule_require_singleuser_auth",
+        "true",
+    ),
```

Comment:
What is the intent behind testing all these rule IDs here?

Reply:
These rules contain all common cases of OVAL trees.